qp_deblur/deblur.py

```diff
@@ -373,6 +373,9 @@ def deblur(qclient, job_id, parameters, out_dir):
     prep_info = qclient.get(
         '/qiita_db/prep_template/%s/' % artifact_info['prep_information'][0])
     df = pd.read_csv(prep_info['prep-file'], sep='\t')
+    if prep_info['data_type'] not in {'16S', '18S', 'ITS'}:
```

Comment:
This variable doesn't look like it exists in any of the AGP preps. Is it common and controlled in Qiita?

Reply:
```python
In [61]: pts = qdb.study.Study(10317).prep_templates()

In [62]: obs = set()
    ...: for pt in pts:
    ...:     df = pt.to_dataframe()
    ...:     if 'data_type' in df.columns:
    ...:         obs.update(set(df['data_type']))
    ...:     else:
    ...:         print("Missing: %d" % pt.id)
... output cut ...
In [63]: obs
Out[63]: set()

```

Reply:
Yup, given by: https://github.com/biocore/qiita/blob/master/qiita_db/handlers/prep_template.py#L77, which is called 2 lines above.

Reply:
Oh I see, thanks!